tex/chapters/gausss-law/infinite-wire.tex

```diff
@@ -0,0 +1,22 @@
+\section{Infinitely long, straight wire}
+Given a wire with charge density per unit of length $\lambda$ in empty space, due to the cylindrical symmetry of the system, the electric 
```

Comment:
You must specify that the wire is along the $z$-axis, otherwise line 3 doesn't make sense.

Reply:
done